src/main/java/org/opensearch/ad/model/AnomalyResult.java

```diff
@@ -316,34 +494,39 @@ public AnomalyResult(StreamInput input) throws IOException {
         this.taskId = input.readOptionalString();
         this.modelId = input.readOptionalString();
 
-        this.totalUpdates = input.readOptionalLong();
         this.startOfAnomaly = input.readOptionalBoolean();
         this.inHighScoreRegion = input.readOptionalBoolean();
-        this.relativeIndex = input.readOptionalInt();
+        // if anomaly is caued by current input, we don't show approximate time
```

Comment:
`caued` -> `caused`

Reply:
can you add a comment on line 124 that this can be `null` if `relativeIndex` is 0?

Reply:
fixed